### DISCUSSION TITLE
Docker multiplatform image

### DISCUSSION THREAD

**Discussion by bracconi** [ghostfolio/ghostfolio#1030] [Ideas]
Currently only amd64 architecture is available for the docker image.
I would love a multi architecture image including arm32 and arm64.
This will allow to selfhost ghostfolio  to a raspberry pi for example.

**Comment by dtslvr** [comment]
Progress: #1130

**Reply by dtslvr** [reply]
Ghostfolio [1.184.2](https://hub.docker.com/r/ghostfolio/ghostfolio/tags) has been published for `linux/amd64` and `linux/arm64`.

**Reply by bracconi** [reply]
Great news, Thomas!
